packages/lime-system/files/etc/config/lime-defaults

```diff
@@ -17,6 +17,7 @@ config lime network
 	option anygw_dhcp_limit '0'
 	option main_ipv6_address '2a00:1508:0a%N1:%N200::/64'
 	option bmx6_mtu '1500'
+	option bmx6_publish_ownip '0'
```

Comment:
I believe it more "self documenting" use the boolean value false instead of 0 (may mislead the user to think it's an integer value)